run-python/python.go

```diff
@@ -0,0 +1,266 @@
+package runner
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/stoic-cli/stoic-cli-core"
+)
+
+const (
+	getPipCacheKey = "run-python/get-pip.py"
+	getPipURL      = "https://bootstrap.pypa.io/get-pip.py"
+
+	pipCacheBase     = ".pip-cache"
+	readyBase        = ".ready"
+	requirementsBase = ".requirements"
+
+	pipRequirements = "" +
```

Comment:
Change to multiline string " -> `

Reply:
I didn't like how I'd have to drop indentation for the multiline string, in order to avoid adding unnecessary whitespace to the string.